content/cds_hooks.md

```diff
@@ -0,0 +1,117 @@
+---
+title: CDS Hooks
+layout: cds_hooks
+---
+
+# CDS Hooks #
+------------------------------------------------------------------------
+<%= beta_tag(resource: false) %>
+* TOC
+{:toc}
+
+## General ##
+
+### Introduction ###
+The CDS Hooks (Clinical Decision Support Hooks) workflow provides external CDS Service developers the ability to send CDS cards to an electronic health record (EHR). The external CDS service will be triggered based on conditions such as opening a patient chart for the specified demographic(age, gender, etc.) or when an order is selected and the cards returned from the service will be presented to the user in the EHR. The cards can contain textual guidance, an actionable suggestion for the user, or a link to launch a SMART app (See [SMART](./smart) for more information).
+
+As a CDS Service Developer, you will work with clients or on your own to create a service that will be called by the CDS Hooks Server when a condition you specify is met. To develop a service, you should first gather the information needed for clients to register your service (See [CDS Service Registration](./cds_hooks/#cds-service-registration)). Then, build your service API so that is uses correct authorization (See [Service Authorization](./cds_hooks/#service-registration)) and returns a properly formatted response (See [Service Response](./cds_hooks/#service-response)). Finally, your service should be available for the CDS Hooks Server to call on a client by client basis.
```

Comment:
I must have forgotten to post this comment. I was thinking may be the below text reads better. what do you think?

`As a CDS Service Developer, you will work with clients or on your own to create a service based on the [CDS Hooks specification](https://cds-hooks.hl7.org/) that will be called by the CDS Hooks Server when a condition you specify is met. Your service API should be build so that it uses correct authorization (See [Service Authorization](./cds_hooks/#service-authorization)) and returns a properly formatted response (See [Service Response](./cds_hooks/#service-response)). Once you are ready to integrate your service to the EHR system you will need to request your service to be registered (See [CDS Service Registration](./cds_hooks/#cds-service-registration)). Finally, your service should be available for the CDS Hooks Server to call on a client by client basis.`

Reply:
https://github.com/cerner/fhir.cerner.com/pull/707/commits/cad04b6feb5cf44d4cd3788114785918a902e83a#diff-179aa7a92baef1c89bbd6fabd30f690e85b09c4e22896697d9d13b17dc80d5e7R17